_docs_integrate/attribute-introduction.md

```diff
@@ -84,51 +89,55 @@ The [RelationshipAttribute value types]({% link _docs_integrate/attribute-values
+The own shared RelationshipAttribute is the LocalAttribute of the `owner` of the RelationshipAttribute and the peer's LocalAttribute is referred to as a peer shared RelationshipAttribute.
 
-Sometimes an own shared RelationshipAttribute is referred to as an own shared ThirdPartyRelationshipAttribute. This is the case if it originates from another own shared RelationshipAttribute that exists in the context of a different Relationship. An own shared ThirdPartyRelationshipAttribute can be identified by the fact that the `id` of the own shared RelationshipAttribute used as the source is stored in its `shareInfo.sourceAttribute` property. This is the case at least until the own shared RelationshipAttribute used as the source has not been [deleted]({% link _docs_integrate/delete-attributes.md %}#delete-own-shared-attributes).
-{: .notice--info}
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/dd79b31c-0404-48bb-9773-9b989801c03c" id="RbDyLSjh7BzY"></iframe></div>
 
-#### Peer shared RelationshipAttributes
+Of course, the `shareInfo` property of the own shared RelationshipAttribute is set. Within the `shareInfo.peer` property, the Address of the peer to whom the RelationshipAttribute's `owner` has established the Relationship is specified.
+If the RelationshipAttribute is initially created for a specific Relationship and does not originate from another RelationshipAttribute, the `shareInfo.sourceAttribute` property is undefined.
+The peer shared RelationshipAttribute is the peer's [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) and forms the counterpart of the own shared RelationshipAttribute. The `id` of a peer shared RelationshipAttribute is always the same as the `id` of the associated own shared RelationshipAttribute. As with the own shared RelationshipAttribute, the `shareInfo` property of a peer shared RelationshipAttribute is of course set.
+Within the `shareInfo.peer` property, the Address of the `owner` of the RelationshipAttribute is specified.
+It should be emphasized that this is characteristic for a peer shared RelationshipAttribute.
+In any case, the `shareInfo.sourceAttribute` property of a peer shared RelationshipAttribute is undefined.
+This is plausible if the RelationshipAttribute is initially created for a specific Relationship and does not originate from another RelationshipAttribute, but this is also the case if it originates from another.
+This is because the LocalAttribute which is associated with the RelationshipAttribute and which is used as the source for sharing can only be available locally to the RelationshipAttribute's sharer.
```

Comment:
So you would prefer to delete the entire paragraph `"This is plausible if the RelationshipAttribute is initially created for a specific Relationship and does not originate from another RelationshipAttribute, but this is also the case if it originates from another.
This is because the LocalAttribute which is associated with the RelationshipAttribute and which is used as the source for sharing can only be available locally to the RelationshipAttribute's sharer."`? Would be okay for me, I just want to make sure that I understand you correctly.

Reply:
Yes. I'd reformulate the sentence "In any case, the `shareInfo.sourceAttribute` property of a peer shared RelationshipAttribute is undefined." In the next section then, you can mention again that the shareInfo.sourceAttribute of a peer shared ThirdPartyRelationshipAttribute is undefined, too, since the peer doesn't own the source Attribute.